src/mono/netcore/CoreFX.issues.rsp

```diff
@@ -1,799 +1,3 @@
-# Expects an ArgumentException
-# https://github.com/mono/mono/issues/15158
--nomethod System.Threading.Tests.MutexTests.OpenExisting_InvalidNames
-
-# Expects ArgumentException but none is thrown
-# https://github.com/mono/mono/issues/15159
--nomethod System.Threading.Tests.MutexTests.Ctor_InvalidNames_Unix
-
-# Expects PlatformNotSupportedException, but we give an ArgumentNullException
-# https://github.com/mono/mono/issues/15160
--nomethod System.Threading.Tests.SemaphoreTests.OpenExisting_NotSupported_Unix
-
-# Expects PlatformNotSupportedException but none thrown
-# https://github.com/mono/mono/issues/15161
--nomethod System.Threading.Tests.SemaphoreTests.Ctor_NamesArentSupported_Unix
-
-####################################################################
-##  System.Threading.Tests
-####################################################################
-
-# Requires precise GC (should be ignored in dotnet/corefx for mono)
--nomethod System.Threading.Tasks.Tests.ExecutionContextFlowTest.TaskCompletionSourceDoesntCaptureExecutionContext
-
-# flaky, causes stack overflow
--nomethod System.Threading.Tasks.Tests.TaskContinueWithTests.LongContinuationChain_ContinueWith_DoesNotStackOverflow
-
-####################################################################
-##  System.Threading.ThreadPool.Tests
-####################################################################
-
-# Tests for ThreadPool.SetMaxThreads(-1, -1) == true, which only happens to work on CoreCLR because
-# the managed SetMaxThreadsNative prototype uses "int"s while the unmanaged code uses "DWORD"s and
-# thus it interprets it as large positive numbers.
-#
-# https://github.com/mono/mono/issues/15164
--nomethod System.Threading.ThreadPools.Tests.ThreadPoolTests.SetMinMaxThreadsTest
-
-# TODO: Differences in behaviour between NetFX and CoreFX
--nomethod System.Threading.ThreadPools.Tests.ThreadPoolTests.SetMinThreadsTo0Test
-
-# Explicitly skipped if it's MonoVM (SkipOnTargetFramework)
--nomethod System.Threading.ThreadPools.Tests.ThreadPoolTests.SetMinMaxThreadsTest_ChangedInDotNetCore
-
-####################################################################
-##  System.ComponentModel.Composition.Tests
-####################################################################
-
--nomethod System.ComponentModel.Composition.CompositionServicesTests.GetDefaultContractNameTest
--nomethod System.ComponentModel.Composition.MetadataViewProviderTests.GetMetadataView_InterfaceWithIndexer_ShouldThrowNotSupportedException
--nomethod Tests.Integration.ExportFactoryTests.ExportFactoryStandardImports_ShouldWorkProperly
-
-# flaky test - System.TypeLoadException : Could not load type '_proxy_System.ComponentModel.Composition.IStronglyTypedStructure_3b7f133d-79c7-449e-8f68-22aa093dbb8d' from assembly ''.
--nomethod System.ComponentModel.Composition.MetadataAttributeTests.StronglyTypedStructureTest
```

Comment:
this test wasn't disabled in the source (it failed in the PR).

Reply:
yeah, seems that it's still flaky. I'll file an issue.